doltpy/cli/write/write.py

```diff
@@ -0,0 +1,172 @@
+from typing import List, Callable, Mapping, Any
+import pandas as pd
+from doltpy.cli import Dolt
+from doltpy.shared.helpers import columns_to_rows
+import logging
+import io
+import tempfile
+import os
+import csv
+from datetime import datetime
+
+logger = logging.getLogger(__name__)
+
+CREATE, FORCE_CREATE, REPLACE, UPDATE = 'create', 'force_create', 'replace', 'update'
+IMPORT_MODES_TO_FLAGS = {CREATE: ['-c'],
+                         FORCE_CREATE: ['-f', '-c'],
+                         REPLACE: ['-r'],
+                         UPDATE: ['-u']}
+
+
+def write_file(dolt: Dolt,
+               table: str,
+               file_handle: io.StringIO,
+               # TODO what to do about this?
+               filetype: str = 'csv',
```

Comment:
different writer function for different files/compression types

Reply:
These just dispatch into the command line, so probably not to be honest.